proto/storage/cluster.proto

```diff
@@ -143,7 +143,7 @@ message SensorDeploymentIdentification {
 message Cluster {
   string id = 1; // @gotags: search:"Cluster ID,hidden,store" sql:"pk,type(uuid)"
   string name = 2; // @gotags: search:"Cluster,store" sql:"unique"
-  ClusterType type = 3;
```

Comment:
Could you pull out the schema change into a separate PR and push it with the respective migration?

Reply:
Done